### ISSUE TITLE
whats cifar_head?

### ISSUE THREAD

**Issue by steveguang** [AndrewAtanov/simclr-pytorch#5]
Hi, @AndrewAtanov . Wondering what cifar_head means. And why is conv1 needed to be added? Can you explain? Thanks!
if cifar_head:
            self.conv1 = nn.Conv2d(3, 64, kernel_size=3, stride=1, padding=1, bias=False)

**Comment by AndrewAtanov** [comment]
Hi @steveguang! The first layer in the standard ResNet50 is 7x7 Conv with stride 2, which would greatly reduce the spatial information of a 32x32 cifar10 image, so it's replaced with 3x3 Conv in the original simclr implementation, which we follow. Refer to Appendix B.9 in [https://arxiv.org/pdf/2002.05709.pdf].
